### DISCUSSION TITLE
appearance-samed app is being sold on Google Play

### DISCUSSION THREAD

**Discussion by suucss** [2dust/v2rayNG#1708] [General]
I was explorering google play a few minutes ago, and found that under the item "v2rayng" an app called "v2fly" is being sold whose appearance looks as same as our app, as the followed pic:
逛 Play 商店，发现有人在卖外观雷同的软件，大概率是直接倒卖
![Screenshot_20221023_175122](https://user-images.githubusercontent.com/84609589/197386063-86a76435-10c0-420b-ac86-de1c8b3fc20c.jpg)

**Comment by cuynu** [comment]
This is an app use for donate developer, the developer of v2rayNG
Its only a v2rayNG app with renamed for donation 
https://github.com/2dust/v2flyNG

**Reply by suucss** [reply]
Sorry about that. I was worried that the app was being sold, and I didn't know that app is a donation version of our app, sorry about that again!
